### DISCUSSION TITLE
Content can only be sent to Obsidian once per browser refresh

### DISCUSSION THREAD

**Discussion by huangjingyun8** [coddingtonbear/obsidian-web#164] [Bug Report]
### Describe the bug

我是保存到 每日笔记的，今天自动更新后，就是一场噩梦

点保存一次后，你再点，就陷入了循环，永远只会出现这个保存成功界面

<img width="600" alt="image" src="https://github.com/coddingtonbear/obsidian-web/assets/56347021/a3687d2d-4367-4d9a-81f8-b65cf1574077">

> (Machine Translation): I saved it to Daily Notes. After the automatic update today, it was a nightmare.
> 
> After clicking save once, if you click again, you will be stuck in a loop, and only this save success interface will always appear.

### To Reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected Behavior

恢复正常，跟之前一样，可以正常保存 即可

### Operating System

mac

### Browser

chrome

### Browser Version

 121.0.6167.160

**Comment by huangjingyun8** [comment]
除非重启浏览器，否则永远是 只出现 all done 页面

**Reply by theotheo** [reply]
At my observations reload the page could help

**Comment by theotheo** [comment]
I've faced with the same issue after automatic update.

**Comment by coddingtonbear** [comment]
~~I haven't seen this issue personally, so I'm afraid I can't be sure, but I did just upload a new package to the Chrome App Store a moment ago that _might_ address the issue once it's approved by the app store approval team (see #163).  That said, given that I can't reproduce this, I really can only guess.~~

Actually, nevermind -- I do see the problem you're describing now and will be updating the text above slightly for searchability.

**Comment by coddingtonbear** [comment]
As a temporary workaround, you _should_ be able to get the window to allow a new submission once again if you refresh the browser window.

**Comment by coddingtonbear** [comment]
I have found a fix for this bug and have committed it to the repository as v3.2.2.  Unfortunately, I cannot yet submit this package for review on the Chrome App Store until the existing bugfix (v3.2.1, see #163) is approved.  Once it is again possible for me to submit updates to the app store, I'll send that one in, too.

**Comment by coddingtonbear** [comment]
& submitted for review!  If the last couple updates take a similar amount of time, we can expect that the new version will be available to Chrome within an hour or two.

**Comment by coddingtonbear** [comment]
Well -- that was quick.  It looks like that update was already approved.

Chrome will update the installed version automatically over time, but if you would like to rush that process along, you can get the latest version by following these instructions: https://github.com/coddingtonbear/obsidian-web/wiki/How-do-I-ask-Chrome-to-update-extensions-immediately%3F

**Comment by coddingtonbear** [comment]
Thanks for the bug report, by the way!  I'm not sure how long it would've taken me to notice this problem without it!

**Comment by huangjingyun8** [comment]
非常感谢，我太喜欢您这个chrome插件，在 ob 生态里，您的解决方案是唯一的，可以完美地将 高亮文本 剪辑到 ob 每日日记的。

非常完美，无以复加的感激
